modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -83,9 +83,9 @@ The *Manage Your Verification Methods* interface appears.
 . Next to the verification method you want to configure, click *Add*.
 . Follow the instructions for your preferred verification method.
 
-== Modify or Remove Multi-Factor Authentication on Your Account
+== Manage Multi-Factor Authentication for Your Account
 
-If you have previously configured multi-factor authentication, you can modify which verification methods you use for your account. If your organization requires multi-factor authentication, you must have at least one verification method configured.
+If you use multi-factor authentication, because your organization requires it or you've turned it on for yourself, you can modify the verification methods that you use to log in. You must register at least one method to use MFA.
```

Comment:
```suggestion
If you use multi-factor authentication because your organization requires it or if you've turned it on for yourself, you can modify the verification methods that you use to log in. You must register at least one method to use MFA.
```